src/utils/index.js

```diff
@@ -0,0 +1,13 @@
+import { BitcoinHelpers } from '@keep-network/tbtc.js'
+
+import BN from "bn.js"
+
+import BigNumber from "bignumber.js"
+BigNumber.set({ DECIMAL_PLACES: 8 })
+
+export function formatSatsToBtc(sats) {
+    sats = BN.isBN(sats) ? sats.toString() : sats
```

Comment:
Is there a case where `sats` isn't a `BN` and `sats.toString()` doesn't do what we want it to? Feels like we could always do a `sats.toString()`.

Reply:
Good call. Will remove